google_gemini/README.md

```diff
@@ -0,0 +1,125 @@
+    -e DD_API_KEY=<DATADOG_API_KEY> \
+    -p 127.0.0.1:8126:8126/tcp \
+    -p 127.0.0.1:8125:8125/udp \
+    -e DD_DOGSTATSD_NON_LOCAL_TRAFFIC=true \
+    -e DD_APM_ENABLED=true \
+    gcr.io/datadoghq/agent:latest
+  ```
+
+2. If you haven't already, install the `ddtrace` package:
+
+  ```shell
+    pip install ddtrace
+  ```
+
+3. Start your application using the `ddtrace-run` command to automatically enable tracing:
+
+  ```shell
+     DD_SITE=<YOUR_DATADOG_SITE> DD_API_KEY=<YOUR_API_KEY> DD_LLMOBS_ENABLED=1 DD_LLMOBS_ML_APP=<YOUR_ML_APP_NAME> ddtrace-run python <YOUR_APP>.py
+  ```
+
+**Note**: If the Agent is running on a custom host or port, set `DD_AGENT_HOST` and `DD_TRACE_AGENT_PORT` accordingly.
+
+##### If you are running LLM Observability in a serverless environment (AWS Lambda):
+1. Install the **Datadog-Python** and **Datadog-Extension** Lambda layers as part of your AWS Lambda setup.
+2. Enable LLM Observability by setting the following environment variables:
+
+  ```shell
+     DD_SITE=<YOUR_DATADOG_SITE> DD_API_KEY=<YOUR_API_KEY> DD_LLMOBS_ENABLED=1 DD_LLMOBS_ML_APP=<YOUR_ML_APP_NAME>
+  ```
+
+**Note**: In serverless environments, Datadog automatically flushes spans when the Lambda function finishes running.
+
+##### Automatic Google Gemini tracing
+
+The Google Gemini integration provides automatic tracing for the Google AI Python SDK's content generation calls. This captures latency, errors, input and output messages, as well as token usage for Google Gemini operations.
+
+The following methods are traced for both synchronous and asynchronous Google Gemini operations:
+- Generating content (including streamed calls): `model.generate_content()`, `model.generate_content_async()`
+- Chat messages: `chat.send_message()`, `chat.send_message_async()`
+
+No additional setup is required for these methods.
+
+##### Validation
+Validate that LLM Observability is properly capturing spans by checking your application logs for successful span creation. You can also run the following command to check the status of the `ddtrace` integration:
+
+  ```shell
+  ddtrace-run --info
+  ```
+
+Look for the following message to confirm the setup:
+
+  ```shell
+  Agent error: None
+  ```
+
+##### Debugging
+If you encounter issues during setup, enable debug logging by passing the `--debug` flag:
+
+  ```shell
+  ddtrace-run --debug
+  ```
+
+This displays any errors related to data transmission or instrumentation, including issues with Google Gemini traces.
+
+## Data Collected
+
+### Metrics
+
+See [metadata.csv][1] for a list of metrics provided by this integration.
+
+### Service Checks
+
+The Google Gemini integration does not include any service checks.
+
+### Events
+
+The Google Gemini integration does not include any events.
+
+## Troubleshooting
+
+Need help? Contact [Datadog support][2].
+
+[1]: https://github.com/DataDog/integrations-core/blob/main/google-gemini/metadata.csv
+[2]: https://docs.datadoghq.com/help/
```

Comment:
```suggestion
[2]: https://docs.datadoghq.com/help/
[3]: https://gemini.google.com/
[4]: https://imgix.datadoghq.com/video/products/llm-observability/expedite-troubleshooting.mp4?fm=webm&fit=max
```

Reply:
I'm not sure the overview has access to the numbered links at the end of the readme docs that is why I put it expanded